### ISSUE TITLE
A lot of syntax errors in exception handling and "print" syntax

### ISSUE THREAD

**Issue by born4web** [google/gdata-python-client#39]
Hello,
I am new here and downloaded this lib today.

During installing there were a lot of syntax errors found in exception raising. And there were some mistakes in some "print" callings...

Now I have version of files with now error during setup. I am not sure if I corrected all errors right according the lines meaning...

So any body interested to look and check my corrections?

Petr

**Comment by aliafshar** [comment]
You are using Python 3, right? This library is Python 2.

**Comment by born4web** [comment]
Yes I do. I try to fight that but...probably wait for new version :-)

Do you plan to use Python 3 in future?

**Comment by aliafshar** [comment]
Python3 is not a high priority in this library, but I see people have had
success porting it.

On Sat, Jan 2, 2016 at 2:52 PM, born4web notifications@github.com wrote:

> Yes I do. I try to fight that but...probably wait for new version :-)
> 
> Do you plan to use Python 3 in future?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/google/gdata-python-client/issues/39#issuecomment-168436191
> .

## 

ali afshar + http://google.com/+alihafshar developer platform engineering
